src/errors/request.js

```diff
@@ -16,7 +16,10 @@ export default class RequestError extends FakeError {
       throw new Error('No url passed to RequestError');
     }
 
-    const message = createMessage(error.status || error.code, url);
+    const message = createMessage(
+      error.status || error.statusCode || error.code || error.body.code,
+      url,
+    );
```

Comment:
We should not be altering the signature of errors. It's not related to this new feature and is a large breaking change.

Reply:
Fixing a bug where a large portion of the time, we weren't extracting the error code correctly, so it would say "undefined" in the error message to the user.

Reply:
Certainly that's not happening in all cases or we would have heard something by now from users. Can you provide examples?